habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/PyenvAndPoetrySetup.java

```diff
@@ -147,6 +160,15 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             log.info("Checking for updates to poetry-monorepo-dependency-plugin...");
             poetryHelper.installPoetryPlugin("poetry-monorepo-dependency-plugin");
         }
+
+        if (StringUtils.isEmpty(username) || StringUtils.isEmpty(password)) {
+            log.info(String.format("Did not find username and password for the <server> with <id> %s. Will pull anonymously", pypiRepoId));
```

Comment:
A: This is a bit of a lie. If poetry already has auth configured for the server then even if the username/password are empty here doesn't mean we'll use anonymous auth.

Reply:
I edited the message in my latest commit.